apps/emqx_bridge_opents/src/emqx_bridge_opents_connector.erl

```diff
@@ -167,9 +167,10 @@ on_batch_query(
     BatchReq,
     #{channels := Channels} = State
 ) ->
+    [{ChannelId, _} | _] = BatchReq,
```

Comment:
are all items in the batch from the same channel ?

Reply:
Yes, the actions don't share `resource_buffer_worker`s.